src/it/test15Implicit/pom.xml

```diff
@@ -108,7 +108,6 @@
 				<version>1.0-SNAPSHOT</version>
 				<executions>
 					<execution>
-						<phase>package</phase>
```

Comment:
You took this out because it was unnecessary?  (since the Mojo impl has a dflt binding to phase already)